.pipelines/pr-e2e.yaml

```diff
@@ -70,94 +70,37 @@ jobs:
     skipTests: 'should have healthy time synchronization'
     stabilityIterations: '10'
 
-- template: e2e-job-template.yaml
-  parameters:
-    name: 'k8s_1_11_release_e2e'
-    k8sRelease: '1.11'
-    apimodel: 'examples/e2e-tests/kubernetes/release/default/definition.json'
-    createVNET: true
-    stabilityIterations: '3'
-
-- template: e2e-job-template.yaml
-  parameters:
-    name: 'k8s_1_12_release_e2e'
-    k8sRelease: '1.12'
-    apimodel: 'examples/e2e-tests/kubernetes/release/default/definition.json'
-    createVNET: true
-    stabilityIterations: '3'
-
 - template: e2e-job-template.yaml
   parameters:
     name: 'k8s_1_13_release_e2e'
     k8sRelease: '1.13'
     apimodel: 'examples/e2e-tests/kubernetes/release/default/definition.json'
     createVNET: true
-    stabilityIterations: '3'
```

Comment:
3 is the default, I assume?

Reply:
So we've been using 3 for the "linux" tests, which are occasionally flaky :(

We were skipping for "windows" tests.

The most cautious way to converge the two was to stop running these "stability" tests on PR tests for now.

(That's my thinking, at least.)